lib/jormungandr/test/data/jormungandr/start_node

```diff
@@ -0,0 +1 @@
+cardano-wallet-jormungandr launch --genesis-block block0.bin --secret secret.yaml
```

Comment:
I suppose this shouldn't be there ;)

Reply:
Actually I've added it here on purpose... just to serve as a helper script...